fiftyone/utils/sam3.py

```
@@ -466,37 +508,60 @@ def collate_fn(batch):
-                            # dummy values
-                            coco_image_id=img_idx,
-                            original_category_id=1,
-                            object_id=0,
-                            frame_index=0,
-                        ),
+                box_classes = results["classes"][img_idx]
+                box_labels = results["boxes_labels"][img_idx]
+
+                # Group boxes by class label
+                class_groups = {}
+                for box, cls, lbl in zip(box_prompts, box_classes, box_labels):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)